### DISCUSSION TITLE
[Question] Roadmap

### DISCUSSION THREAD

**Discussion by CorruptionVR** [yume-chan/tango-feedback#17] [Q&A]
hey, do you have a roadmap for features you plan to add to the new web app? if so, could you please put it in the README or an announcement so that people don't request features that you are planning to add?

**Comment by yume-chan** [comment]
I'm working on:

* #4
* Share device using account. (Similar to the existing device sharing feature, but by logging into the same account on both PC, devices connected to one PC will automatically be available on the other PC). This will be a paid feature, because of extra storage server cost.
* Mobile browser optimization

Other than those, I don't have more planned features. I also want to focus on features users actually need, so suggestions are welcomed.
